.github/pull_request_template.md

```diff
@@ -25,5 +25,17 @@ Please include any screenshots that communicate the visual story of the change t
 ### Notes
 
 <!--
-Any notes that might be helpful for a reviewer to keep in mind while reading the changes.
+Please include any notes that might be helpful for a reviewer to keep in mind while reading the changes.
 -->
+
+### Optional Tasks
+
+<!--
+Common, optional tasks are included here in case you forgot something important.
+-->
+
+- [ ] Include 🎩  Instructions
+- [ ] Update the readme (README.md)
+- [ ] Update the API or architecture docs (e.g. docs/api.md)
+- [ ] Increment the changelog (CHANGELOG.md)
+- [ ] Increment the version number (lib/version.rb)
```

Comment:
If we're gearing this for proper Gem support then the following step must be added:

```
- [ ] Create a release in GitHub
```

Or at the very least mentioned somewhere so people don't forget and muck up the changelog for a gem.